src/utils/base64.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Punica - LwM2M server with REST API
+ * Copyright (C) 2018 8devices
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef PUNICA_UTILS_BASE64_H
+#define PUNICA_UTILS_BASE64_H
+
+#include <stdint.h>
+
+size_t base64_encoded_length(size_t data_length);
+size_t base64_decoded_length(size_t b64_string_length);
+
```

Comment:
Previous headers had an API description. It would be nice to have not decreasing documentation and preferably increasing :)